buddyboss/buddyboss-platform
# PROD-1410 #414-Theme-issue Fix PHP: Add Filter to add style to link tag

### All Submissions:






### Changes proposed in this Pull Request:



Fixes https://github.com/buddyboss/buddyboss-theme/issues/414

### How to test the changes in this Pull Request:

1. Go to forum pages 
2. Check the sub forum styles are consistent.

### Other information:
